Darkly/LDUserBuilder.m

```diff
@@ -228,8 +235,22 @@ -(User *)build
         [user setAnonymous:currentAnonymous];
     }
     DEBUG_LOG(@"LDUserBuilder building User with anonymous: %d", anonymous);
-    [[DataManager sharedManager] saveContext];
+    [self saveUser: user];
     return user;
 }
 
+-(void) saveUser: (User *) user {
```

Comment:
Since this method is just working with CoreData, should it exist here or in DataManager?